SQL/0000-00-00-schema.sql

```diff
@@ -754,6 +757,40 @@ CREATE TABLE `mri_protocol_checks` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
 
+-- ********************************
+-- BIDS tables
+-- ********************************
+
+CREATE TABLE `bids_category` (
+ `BIDSCategoryID` int(3) NOT NULL AUTO_INCREMENT,
+ `ImagingCategory` varchar(10) NOT NULL UNIQUE,    
+ PRIMARY KEY (`BIDSCategoryID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO `bids_category` (ImagingCategory) VALUES
+      ('anat'),
+      ('func'),
+      ('dwi'),
+      ('fmap');
+
+CREATE TABLE `bids_mri_scan_type_rel` (
+  `MRIScanTypeID` int(10) unsigned NOT NULL,
+  `BIDSCategory`varchar(10) DEFAULT NULL,
```

Comment:
This should be int(3) and link to the BIDSCategoryID not ImagingCategory. 

Sync update script below.